editor/store/selectors.js

```diff
@@ -1584,32 +1615,3 @@ export function inSomeHistory( state, predicate ) {
 export function getBlockListSettings( state, uid ) {
 	return state.blockListSettings[ uid ];
 }
-
-/**
- * Determines the blocks that can be nested inside a given block. Or globally if a block is not specified.
- *
- * @param {Object}           state                     Global application state.
- * @param {?string}          uid                       Block UID.
- * @param {string[]|boolean} globallyEnabledBlockTypes Globally enabled block types, or true/false to enable/disable all types.
- *
- * @return {string[]|boolean} Blocks that can be nested inside the block with the specified uid, or true/false to enable/disable all types.
- */
-export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
```

Comment:
Should it also be deprecated for some time?

Reply:
Yes—good catch 😄